modules/examples/flake/flake.nix

```diff
@@ -27,6 +27,9 @@
       programs.zsh.enable = true;  # default shell on catalina
       # programs.fish.enable = true;
 
+      # Set Git commit hash for darwin-version.
+      system.configurationRevision = self.rev or self.dirtyRev or null;
```

Comment:
Needing this boilerplate in every flake is unfortunate... but I don't presently see any way around it, so it's probably fine.